README.md

```diff
@@ -1,4 +1,21 @@
 # Simul
+
+What is Simul?
+    From: https://github.com/LLNL/simul
+
+    "simul" is an MPI coordinated test of parallel filesystem system calls and
+    library functions.  It was designed to perform filesystem operations
+    simultaneously from many nodes and processes to test the correctness
+    and coherence of parallel filesystems.
+
+How is simul used in DAOS?
+
+    Simul can let us know what POSIX operations are working or not
+    in a POSIX container.
+
+    There should be an automated test for this.
```

Comment:
What do you mean by there should be a test? Is it going to be implemented in the future version?

Reply:
Yes, in fact is here: https://github.com/daos-stack/daos/pull/5958 but I need the simul package installed on the test nodes before running it

Reply:
I see. Then you might want to mention that because it sounds like you're still in middle of your work.